lib/src/table/table.component.scss

```diff
@@ -0,0 +1,44 @@
+@import "~@sebgroup/bootstrap/scss/variables";
+
+.seb-table {
+    &.fixed-height {
+        height: 200px;
+        overflow: auto;
+        width: 100%;
+        margin-bottom: 1rem;
+
+        table {
+            margin-bottom: 0px;
+        }
+    }
+
+    table {
+        border-collapse: separate !important; /* Don't collapse */
```

Comment:
i dont get the relation between dont collapse and separate css value, also having an !important there will make life hard for whoever wants to change that style

Reply:
yeah this was left over from before SEB bootstrap had the correct styles. Thanks for spotting this, we don't need it, getting rid of it